car/car.ino

```diff
@@ -3,51 +3,47 @@
 #include <VL53L0X.h>
 #include <Wire.h>
 
-
-int trigPinFront = 19; //D19
-int echoPinFront = 5; //D5
-int MAX_DISTANCE = 300;
-
-int trigPinRight = 33; //D33
-int echoPinRight = 18; //D18
-
-const auto pulsesPerMeter = 600;
-const int TURN_ANGLE = 80;
-const int REVERS_SPEED = 40;
-const int GYRO_OFFSET = 22;
-
+//Ultrasonic sensors 
+const int MAX_DISTANCE = 300;
```

Comment:
3 meters is a really really long distance for these sensors. I have found them to be practically unusable when in motion and when tasked to detect obstacles that will not be at a 90 degrees angle compared to the beam they emit.
Therefore, I would strongly advice to use the smallest distance that makes sense for your application, or the default value (which is 100). In other words, unless you need to, don't pass a maximum distance argument yourselves.